columnflow/calibration/cms/egamma.py

```diff
@@ -300,52 +310,67 @@ def call_func(self, events: ak.Array, **kwargs) -> ak.Array:
             )
 
         # the correction tool only supports flat arrays, so convert inputs to flat np view first
-
         sceta = flat_np_view(events[self.source_field].superclusterEta, axis=1)
         r9 = flat_np_view(events[self.source_field].r9, axis=1)
         flat_seeds = flat_np_view(events[self.source_field].deterministic_seed, axis=1)
 
-        # prepare arguments
-        # we use pt as et since there depends in linear (following the recoomendations)
+        # we use pt as et since there depends in linear (following the recommendations)
         # (energy is part of the LorentzVector behavior)
+        pt = flat_np_view(events[self.source_field].rawPt, axis=1)
+
+        # prepare arguments
         variable_map = {
+            "AbsScEta": np.abs(sceta),
             "eta": sceta,
             "r9": r9,
+            "pt": pt,
             **self.resolution_cfg.corrector_kwargs,
         }
+
         args = tuple(
-            variable_map[inp.name] for inp in self.resolution_corrector.inputs
+            variable_map[inp.name]
+            for inp in self.resolution_corrector.inputs
```

Comment:
If you make this change here, then please also do it in the other call func's in this file.